runelite-client/src/main/java/net/runelite/client/plugins/itemcharges/ItemChargeConfig.java

```diff
@@ -348,7 +373,7 @@ default boolean showInfoboxes()
 		keyName = "showPotionDoseCount",
 		name = "Potion Doses",
 		description = "Show remaining potion doses",
-		position = 25,
+		position = 27,
```

Comment:
You must also increment the position of other `chargesSection` items such as the Bracelet of Slaughter, Expeditious Bracelet, and Guthix Rests to ensure the order is maintained.